ewcmd.py

```diff
@@ -376,7 +376,7 @@ def gen_data_text(
 	
 	# Send the response to the player.
 	resp_cont.format_channel_response(cmd.message.channel.name, cmd.message.author)
-	await resp_cont.post()
+	await resp_cont.post(channel=cmd.message.channel)
```

Comment:
EwResponseContainer doesn't accept a channel argument

Reply:
i changed the function so it does. if the channel argument is the default none, the function runs like usual, but if that argument is passed it overrides the channel with the passed in one. this lets data work in dms

Reply:
oh wait, i didn't push ewutils because debug stuff was on it. my bad.